build-system/release-tagger/utils.js

```diff
@@ -158,19 +158,50 @@ async function getPullRequests(shas) {
 
 /**
  * Get pull requests between two commits
- * @param {string} commit
- * @param {string} previousCommit
+ * @param {string} head
+ * @param {string} base
  * @return {Promise<Array<GraphQlQueryResponseData>>}
  */
-async function getPullRequestsBetweenCommits(commit, previousCommit) {
-  const {commits} = await compareCommits(previousCommit, commit);
+async function getPullRequestsBetweenCommits(head, base) {
+  const {commits} = await compareCommits(base, head);
```

Comment:
Do you think it's worth using the same parameter order (`base, head` vs. `head, base`) for these two functions to prevent bugs from future maintainers who may not notice the flipped order?

Reply:
Sure, makes sense. The `compareCommits` API has it flipped, but future devs won't need to worry about that.